ui/src/keyboardLayouts.ts

```diff
@@ -27,5 +27,6 @@ import { fr_FR } from "@/keyboardLayouts/fr_FR"
 import { it_IT } from "@/keyboardLayouts/it_IT"
 import { nb_NO } from "@/keyboardLayouts/nb_NO"
 import { sv_SE } from "@/keyboardLayouts/sv_SE"
+import { sv_SE } from "@/keyboardLayouts/da_DK"
```

Comment:
this should be `import { da_DK } from "@keyboardLayouts/da_DK" (you didn't change the imported locale)